Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -2661,11 +2661,19 @@ function Set-CustomDRS {
         [Parameter(Mandatory = $true)]
         [String]$ClustersToChange,
         [Parameter(Mandatory = $true,
-            HelpMessage = "The DRS setting. Default of 3 or more conservative of 4.")]
+            HelpMessage = "The DRS setting. Default of 3 or more conservative of 2 or less conservative 4.")]
         [ValidateRange(3, 4)]
-        [int] $Drs
+        [int] $DrsChoice
```

Comment:
Renaming a required parameter is a breaking change, need to either bump the major or keep the name.

Reply:
Sorry -- Didn't know. I've changed them required param back.